Module can't be casted to `typeof request` in 3.9.1

It seems like `teenyRequest` doesn't have the same interface as `request` in 3.9.1, according to TypeScript:

```
src/trace-writer.ts:83:26 - error TS2352: Conversion of type 'typeof teenyRequest' to type 'RequestAPI<Request, CoreOptions, RequiredUriUrl>' may be a mistake because neither type sufficiently overlaps with the other. If this was intentional, convert the expression to 'unknown' first.
  Property 'get' is missing in type 'typeof teenyRequest'.

83           requestModule: teenyRequest as typeof r,
```

I didn't observe this with 3.9.0.

I can help fix this, but given the recency of the changes it seems like @kinwa91 or @JustinBeckwith might have a better idea of where the fix is?

--- a/src/index.ts
+++ b/src/index.ts
@@ -1,6 +1,6 @@
 import * as r from 'request';  // Only for type declarations
 import fetch, * as f from 'node-fetch';
-import {PassThrough} from 'stream';
+import {PassThrough, Stream} from 'stream';
 import * as uuid from 'uuid';
 
 // tslint:disable-next-line variable-name
@@ -111,10 +111,10 @@ function createMultipartStream(boundary: string, multipart: r.RequestPart[]) {
   return stream;
 }
 
-function teenyRequest(reqOpts: r.Options): PassThrough;
+function teenyRequest(reqOpts: r.Options): Stream;
 function teenyRequest(reqOpts: r.Options, callback: r.RequestCallback): void;
-function teenyRequest(
-    reqOpts: r.Options, callback?: r.RequestCallback): PassThrough|void {
+function teenyRequest(reqOpts: r.Options, callback?: r.RequestCallback): Stream|
+    void {
   const {uri, options} = requestToFetchOptions(reqOpts);
 
   const multipart: r.RequestPart[] = reqOpts.multipart as r.RequestPart[];
@@ -249,14 +249,13 @@ function teenyRequest(
 }
 
 teenyRequest.defaults = (defaults: r.OptionalUriUrl) => {
-  return (reqOpts: r.Options, callback?: r.RequestCallback): PassThrough|
-      void => {
-        const opts = {...defaults, ...reqOpts};
-        if (callback === undefined) {
-          return teenyRequest(opts);
-        }
-        teenyRequest(opts, callback);
-      };
+  return (reqOpts: r.Options, callback?: r.RequestCallback): Stream|void => {
+    const opts = {...defaults, ...reqOpts};
+    if (callback === undefined) {
+      return teenyRequest(opts);
+    }
+    teenyRequest(opts, callback);
+  };
 };
 
 export {teenyRequest};

